jetson/percep/pcl.cpp

```diff
@@ -3,19 +3,19 @@
 
 #if OBSTACLE_DETECTION
 /* --- Pass Through Filter --- */
-//Filters out all points with z values that aren't within a threshold
-//Z values are depth values in mm
+//Filters out all points on a given axis that aren't within a threshold
+//Values are depth values in mm
 //Source: https://rb.gy/kkyi80
```

Comment:
What if we made this one multiline comment instead of a mix of multiline and multiple one liners. I.e.:
```
/* Pass Through Filter
 * Filters out ...
 * ...
 */

Reply:
We've formatted all of our code with the current syntax